in_toto/runlib.py

```diff
@@ -880,6 +881,22 @@ def in_toto_record_stop(step_name, product_list, signing_key=None,
     metadata_directory (optional): A directory path to write the resulting link
         metadata file to. Default destination is the current working directory.
 
+    command (optional): A list consisting of a command and arguments executed
+        between in_toto_record_start() and in_toto_record_stop() to capture
+        the command ran in the resulting link metadata.
+
+    byproducts (optional): A dictionary that lists byproducts of the link
+        command execution. It should have at least the following entries
+        "stdout" (str), "stderr" (str) and "return-value" (int).
+
+    environment (optional): A dictionary to capture information about
+        the environment to be added in the resulting link metadata eg.::
```

Comment:
```suggestion
        the environment to be added in the resulting link metadata eg.::

```
Minor nit: sphincs requires a newline here, otherwise there's this error in the doc build log
```
docstring of in_toto.runlib.in_toto_record_stop:56: ERROR: Unexpected indentation.
```

Interestingly enough, the html build still finishes and the code block is rendered correctly. 🤷